install/kubernetes/Makefile.values

```diff
@@ -25,6 +28,10 @@ else
     export HUBBLE_RELAY_REPO:=${CI_REGISTRY}/${CI_ORG}/hubble-relay-ci
 endif
 
+ifndef CILIUM_BRANCH
+$(error "CILIUM_BRANCH needs to be defined")
+endif
+
```

Comment:
Same thing here, we don't need to make this change. See https://github.com/cilium/cilium/pull/32776

Reply:
ohh finally. I didn't know about #32776